guessing-game.html

```diff
@@ -1,21 +1,47 @@
 <script>
-var guestion1;
-var rightAnswer1;
-var wrongAnswer;
+var question1,
+    rightAnswer1,
+    wrongAnswer,
+    question2,
+    rightAnswer2,
+    wrongAnswer2;
 
 wrongAnswer = ("sandwiches");
```

Comment:
since there are a lot of wrong answers, maybe `wrongPluralization` as the variable name instead...?